cognite/client/_api/raw.py

```diff
@@ -495,12 +495,9 @@ def retrieve(self, db_name: str, table_name: str, key: str) -> Row | None:
     def _make_columns_param(self, columns: list[str] | None) -> str | None:
         if columns is None:
             return None
-        if not isinstance(columns, List):
+        if not isinstance(columns, list):
             raise ValueError("Expected a list for argument columns")
-        if len(columns) == 0:
-            return ","
-        else:
-            return ",".join([str(x) for x in columns])
+        return "," if len(columns) == 0 else ",".join([str(x) for x in columns])
```

Comment:
Lgtm, I would switch it around though, also don't need to build a temporary list:
```suggestion
        return ",".join(str(x) for x in columns) if columns else ","
```

Reply:
As a side note, I generally dislike conditional expressions (except when they are super-short) 😉

Reply:
No worries, I will take a mix of both of your comments (removing the list instantiation and returning to the if-else branch).